monkestation/code/modules/power/singularity/spatial_rift.dm

```diff
@@ -0,0 +1,82 @@
+/// spatial rift
+/// Basically a BoH Tear, but weaker because it spawns after nullifying a tesloose or singlo and those have done enough damage
+/obj/singularity/spatial_rift
+	name = "a small tear in the fabric of reality, a good place to stuff problems"
+	desc = "Your own comprehension of reality starts bending as you stare this."
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "boh_tear"
+	is_real = FALSE
+	pixel_x = -32
+	pixel_y = -32
+	dissipate = 0
+	move_self = 0
+	consume_range = 1
+	grav_pull = 2
+	current_size = STAGE_FIVE
+	allowed_size = STAGE_FIVE
+	var/ghosts = list()
+	var/old_loc
+
+/obj/singularity/spatial_rift/Initialize(mapload)
+	. = ..()
+	old_loc = get_turf(src)
+	addtimer(CALLBACK(src, /atom/movable.proc/moveToNullspace), 5 SECONDS) // vanishes after 5 seconds
+	QDEL_IN(src, 10 MINUTES)
+
+/// Retrieve all the items consumed
+/obj/singularity/spatial_rift/proc/retrieve_consumed_items()
+	for(var/atom/movable/content in contents)
+		content.forceMove(old_loc)
+		if(ismob(content))
+			continue
```

Comment:
Won't this continue cause it to skip mobs?

Reply:
Tested it and I can retrieve mobs including myself with the continue and without the continue, so I think it might not make a difference. I'll just remove it anyways.